best-practices/pull-requests.md

```diff
@@ -30,7 +30,7 @@ In order for a PR to be merged, everyone who has been tagged as a reviewer must
 
 ## PR scope and structure
 
-The scope and structure for a PR should be based on the guiding principles that apply to [commit and history structure](https://ably.atlassian.net/l/c/7QP1L31X): PRs should ideally cover a self-contained set of changes, with an overall scope that is large enough that the overall intent of the changes is clear, and small enough that it is manageable to be reviewed in detail. PRs that are too large tend to result in reviews that are more superficial and less effective.
+The scope and structure for a PR should be based on the guiding principles that apply to [commit and history structure](commits.md): PRs should ideally cover a self-contained set of changes, with an overall scope that is large enough that the overall intent of the changes is clear, and small enough that it is manageable to be reviewed in detail. PRs that are too large tend to result in reviews that are more superficial and less effective.
 
 The use of branches generally should follow the policy for [development flow](https://ably.atlassian.net/wiki/spaces/PUB/pages/803766520). If the changes being made are necessarily extensive, then the preferred way to structure the work is as a series of PRs made against an integration branch, so each individual PR remains a manageable size.
```

Comment:
```suggestion
The use of branches generally should follow the policy for [development flow](development-flow.md). If the changes being made are necessarily extensive, then the preferred way to structure the work is as a series of PRs made against an integration branch, so each individual PR remains a manageable size.
```

Reply:
`https://ably.atlassian.net/wiki/spaces/PUB/pages/803766520` link is also broken. Searching "development flow" leads to [this page](https://ably.atlassian.net/wiki/spaces/ENG/pages/311951539/Development+flow+for+Ably+repos), which suggests to use [development-flow.md](https://github.com/ably/engineering/blob/main/best-practices/development-flow.md) from this repo.

Reply:
Thanks! Updated those links.